mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/ReIndexerPerformer.java

```diff
@@ -143,7 +188,7 @@ public ReIndexerPerformer(MailboxManager mailboxManager,
         return mailboxSessionMapperFactory.getMailboxMapper(mailboxSession)
             .findMailboxByIdReactive(mailboxId)
             .flatMap(mailbox -> fullyReadMessage(mailboxSession, mailbox, uid)
-                .flatMap(message -> reIndexMessage(mailboxSession, mailbox, reprocessingContext, message)))
+                .flatMap(message -> reIndexMessage(Either.right(new ReIndexingEntry(mailbox, mailboxSession, message)), reprocessingContext)))
```

Comment:
Can we extract `Either.right(new ReIndexingEntry(mailbox, mailboxSession, message)` in a separate map step to enhance readability?